e2e/components/Tabs/Tabs-test.avt.e2e.js

```diff
@@ -0,0 +1,215 @@
+  }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-tabs--contained-with-secondary-labels',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
+      'Tabs-contained-with-secondary-labels'
+    );
+  });
+
+  test('accessibility-checker tabs contained with secondary labels and icons', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--contained-with-secondary-labels-and-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
+      'Tabs--contained-with-secondary-labels-and-icons'
+    );
+  });
+
+  test('accessibility-checker tabs dismissable', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--dismissable',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--dismissable');
+  });
+
+  test('accessibility-checker tabs dismissable with icons', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--dismissable-with-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--dismissable-with-icons');
+  });
+
+  test('accessibility-checker tabs icon 20 only', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--icon-20-only',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--icon-20-only');
+  });
+
+  test('accessibility-checker tabs icon only', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--icon-only',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--icon-only');
+  });
+
+  test('accessibility-checker tabs manual', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-tabs--manual',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--manual');
+  });
+
+  test('accessibility-checker tabs skeleton', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--skeleton',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs-skeleton');
+  });
+
+  test('accessibility-checker tabs components tabs with icons', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'Tabs',
+      id: 'components-tabs--with-icons',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('Tabs--with-icons');
+  });
+
+  test.slow('default state - keyboard nav', async ({ page }) => {
```

Comment:
We should have another test ensuring dismissable tabs have arrow key navigation and can be dismissed with `Delete`. We should also have another test suite that captures the differences between `automatic` and `manual` (arrow only moves focus, space bar / enter changes tabs) selection modes